apps/journeys-admin/src/components/Editor/Slider/Content/Canvas/InlineEditWrapper/useOnClickOutside/useOnClickOutside.tsx

```diff
@@ -9,12 +9,12 @@ export function useOnClickOutside<T extends HTMLElement = HTMLElement>(
   useEffect(() => {
     const handleClick = (event: MouseEvent): void => {
       const elementClicked = event.target as HTMLElement
-      const canvasClicked =
-        elementClicked.classList.contains('EditorCanvas') ?? false
+      const desktopClicked = elementClicked.classList.contains('EditorCanvas')
+      const mobileClicked = elementClicked.classList.contains('CanvasStack')
+      const canvasClicked = desktopClicked || mobileClicked
       // Prevent double callback triggering
       const inputSelected =
-        elementRef.current?.classList.contains('Mui-focused') ?? false
-
+        elementRef.current?.classList.contains('Mui-focused')
```

Comment:
This one needs to be revered since `elementRef.current` is nullable.
```suggestion
        elementRef.current?.classList.contains('Mui-focused') ?? false
```